testing/scenario_app/lib/src/platform_view.dart

```diff
@@ -765,6 +765,84 @@ class TwoPlatformViewsWithOtherBackDropFilter extends Scenario with _BasePlatfor
   }
 }
 
+/// Builds a scenario where many platform views are scrolling and pass under a widget
+class PlatformViewScrollingUnderWidget extends Scenario with _BasePlatformViewScenarioMixin {
+  /// Creates the PlatformView scenario.
+  ///
+  /// The [dispatcher] parameter must not be null.
+  PlatformViewScrollingUnderWidget(PlatformDispatcher dispatcher, {required this.minId, required this.maxId})
+      : assert(dispatcher != null),
+        super(dispatcher) {
+    for (int i = minId; i < maxId+1; i ++) {
+      createPlatformView(dispatcher, 'platform view', i);
+    }
+  }
+
+  /// The platform view identifier to use for the first platform view.
+  final int minId;
+
+  /// The platform view identifier to use for the last platform view.
+  final int maxId;
+
+  double offset = 0;
+
+  bool movingUp = true;
+
+  @override
+  void onBeginFrame(Duration duration) {
+    _buildOneFrame(offset);
+  }
+
+  @override
+  void onDrawFrame() {
```

Comment:
Have you found an minimal sequence of events that yields into the issue? If possible, it would be preferred.